### PR TITLE
Fix '--where' experiment index filter on info

### PR DESCRIPTION
**Pull request by dapomeroy**
Fixes a bug where filtering using '--where' on the experiment index does not work correctly

### CODE REVIEW

**Review by linsword13 — COMMENTED** [review]
Good stuff! Wonder if we can add a test for it?

**Review by linsword13 — APPROVED** [review]

File: lib/ramble/ramble/cmd/workspace.py [hunk 1]
```diff
@@ -662,6 +662,18 @@ def workspace_info(args):
...
                 print_experiment_set.build_experiment_chains()
 
+                # Reindex the experiments in the print set to match the overall set
+                for exp_name, _, _ in print_experiment_set.all_experiments():
+                    app_inst = experiment_set.get_experiment(exp_name)
+                    experiment_index = app_inst.expander.expand_var_name(
+                        app_inst.keywords.experiment_index
+                    )
+
+                    print_app_inst = print_experiment_set.get_experiment(exp_name)
```
Comment (douglasjacobsen) [inline lib/ramble/ramble/cmd/workspace.py:672]: The second argument in the loop is the same as the print_app_inst. So, I would suggest we change the loop to:

`for exp_name, print_app_inst, _ ...`

and then removing this line.

**Review by douglasjacobsen — CHANGES_REQUESTED** [review]
LG One small request, but otherwise this is great!

File: lib/ramble/ramble/cmd/workspace.py (hunk 1 above)
Comment (dapomeroy) [reply lib/ramble/ramble/cmd/workspace.py:672]: Created https://github.com/GoogleCloudPlatform/ramble/pull/644 to fix this

**Pull request merged**